.circleci/config.yml

```diff
@@ -1243,7 +1243,7 @@ jobs:
       - image: cimg/node:16.14.2
         environment:
           CI: "true"
-    resource_class: large
+    resource_class: xlarge
```

Comment:
🙏

Reply:
I upgraded these tests from medium to large a few months ago, hopefully we don't keep having to do this!

Reply:
I think we can mess with jest config to limit it in how many tests it runs concurrently. but if this is the only issue it's a cheap fix :)